src/main/java/gyro/azure/compute/VirtualMachineResource.java

```diff
@@ -848,4 +928,44 @@ private String getEncodedCustomData() {
     private String getDecodedCustomData(String data) {
         return !ObjectUtils.isBlank(data) ? new String(Base64.getDecoder().decode(data.getBytes())) : null;
     }
+
```

Comment:
Please add a comment line that clearly indicates the following methods are for `GyroInstance` Support.